server/index.js

```diff
@@ -78,11 +78,10 @@ app.post('/api/auth/sign-in', (req, res, next) => {
 
 app.use(authorizationMiddleware);
 
-app.post('/api/inventory/:userId', uploadsMiddleware, (req, res, next) => {
+app.post('/api/inventory', uploadsMiddleware, (req, res, next) => {
   const {
-    articleTypeId, primaryColor, secondaryColor, colorCategoryId, secondaryColorCategoryId
+    userId, articleTypeId, primaryColor, secondaryColor, colorCategoryId, secondaryColorCategoryId
```

Comment:
The userId should come from req.user so that the client is forced to include their token.